templates/security/certifications.html

```diff
@@ -263,7 +282,7 @@ <h3>How do I comply with PCI-DSS?</h3>
       <p>
         Achieving and maintaining compliance is a complex and costly process that involves business processes in addition to software requirements. Ubuntu by Canonical contains software and security controls, such as disk encryption, password settings configuration, <a href="/security/fips">cryptographic compliance with FIPS140-2</a>, <a href="/security/cis">CIS hardening</a> as well as <a href="/pro">a comprehensive Enterprise software maintenance program</a>, to achieve and maintain compliance with the standard.
       </p>
-      <a href="/security/contact-us"
+      <a href="/support#get-in-touch"
```

Comment:
Same here, if we are using a JS modal the `href` should link to a static page

Reply:
Yeah that is true, but copydoc has the other link, what would be better to do here?

Reply:
That is true and we try to always follow the copydoc, but in this case it doesn't make sense. This link will only be accessed if the JS fails for some reason, and if the JS fails here it will likely fail on that page also. '/support#get-in-touch' is also a JS modal

Reply:
Thanks for this input